packages/babel-traverse/src/scope/index.ts

```diff
@@ -227,7 +226,14 @@ function gatherNodeParts(node: t.Node, parts: NodePart[]) {
   }
 }
 
-//
+function resetScope(scope: Scope) {
+  scope.references = Object.create(null);
+  scope.bindings = Object.create(null);
+  scope.globals = Object.create(null);
+  scope.uids = Object.create(null);
+  scope.data = Object.create(null);
+}
+
```

Comment:
This helper seems to be a genuine scope method, I think it could be public as well since the `scope.data` is user defined: They might want to override the `resetScope` method to do custom data cleanup.

Reply:
I prefer not to expose it to users, since we've reduced public methods as much as possible in Babel 8, and they couldn't be overridden before.

Reply:
I rethought it, it seems we shouldn't reset `data`.